sources/packages/web/src/views/aest/student/applicationDetails/StudentAppealRequestsApproval.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <student-appeal-requests-approval
```

Comment:
full page container?

Reply:
its is used inside the component, 
![image](https://github.com/bcgov/SIMS/assets/77353155/453b1f45-c083-4bd7-97ee-ad8fd900cdb9)